tests/e2e/specs/content-helper/related-top-posts-panel-filters.spec.ts

```diff
@@ -11,7 +11,6 @@ import {
 import {
 	VALID_API_SECRET,
 	getTopRelatedPostsMessage,
-	insertRecordIntoTaxonomy,
```

Comment:
Although `insertRecordIntoTaxonomy()` was being used only here, we haven't removed it since it could be of use in the future.